GTG/gtk/editor/taskview.py

```diff
@@ -463,6 +463,37 @@ def insert_tag(self, tag):
         line_mark = self.buff.create_mark("lastline", lastline, False)
         self.insert_at_mark(self.buff, line_mark, tag)
 
+    def remove_tag_editor(self, tagname):
```

Comment:
Choose a better, consistent name. When the method to add tags is `insert_tags`, the method to remove tags should be called `remove_tags` (and accept multiple tags at the same time)